lib/sequent/migrations/view_schema.rb

```diff
@@ -292,7 +291,7 @@ def ensure_valid_plan!
       def migrate_metadata_tables
         Sequent::ApplicationRecord.transaction do
           in_view_schema do
-            exec_sql([ReplayedIds.migration_sql, Versions.migration_sql].join("\n"))
+            exec_sql([Versions.migration_sql].join("\n"))
```

Comment:
Should we not drop the `ReplayedIds` in the last migration?

Reply:
I've added a note to the changelog that this table can be dropped. We could also add a `DROP TABLE IF EXISTS ...` to Sequent but that will have to stay there "forever".